research/object_detection/meta_architectures/context_rcnn_lib_v1.py

```diff
@@ -0,0 +1,225 @@
+# Lint as: python3
```

Comment:
Can you leave this unchanged so we avoid a large diff ? 
You can send a follow up PR with this file renamed.